conan/tools/layout/__init__.py

```diff
@@ -14,3 +14,4 @@ def basic_layout(conanfile, src_folder=".", build_folder=None):
     conanfile.folders.generators = os.path.join(conanfile.folders.build, "conan")
     conanfile.cpp.build.bindirs = ["."]
     conanfile.cpp.build.libdirs = ["."]
+    conanfile.cpp.source.includedirs = ["include"]
```

Comment:
If someone didn't expect this to be set with this layout and set it before calling it to something different, that would now be ignored no?